<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>221</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget_imped" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>531</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget_ekg" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>531</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget_tem" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>390</y>
      <width>531</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
